src/cmdlib.sh

```diff
@@ -37,10 +37,10 @@ arch=$(uname -m)
 export arch
 
 case $arch in
-    "x86_64")  DEFAULT_TERMINAL="ttyS0"   ;;
-    "ppc64le") DEFAULT_TERMINAL="hvc0"    ;;
-    "aarch64") DEFAULT_TERMINAL="ttyAMA0" ;;
-    "s390x")   DEFAULT_TERMINAL="ttysclp0";;
+    "x86_64")  DEFAULT_TERMINAL="ttyS0,115200n8" ;;
```

Comment:
Yeah, it could speed up some parts of the pipeline, specifically, running things in runvm and the live environments of testiso tests (before they reboot into the installed system). It shouldn't change anything else.